public/app/features/explore/ExplorePage.tsx

```diff
@@ -55,7 +56,11 @@ export default function ExplorePage(props: GrafanaRouteComponentProps<{}, Explor
 
   useEffect(() => {
     keybindings.setupTimeRangeBindings(false);
-  }, [keybindings]);
+
+    keybindings.bindWithExploreId('shift+enter', (exploreId) => {
+      dispatch(runQueries({ exploreId }));
+    });
```

Comment:
If I am typing inside an input in a query editor and hit this shortcut it not execute a query with the change I have just made (most editors only commit the model change on blur / enter)